### ISSUE TITLE
Use configuration for externalDependencies

### ISSUE THREAD

**Issue by fbaumgardt** [alpheios-project/arethusa#753]
The locations for modules' JS dependencies are hardcoded in the externalDependencies property, read by the plugins module and subsequently loaded. This breaks when using Arethusa as a plugin, we can

a) hardcode CDN locations
b) move the locations into the config files

I'd prefer b), but have to check the timeline during init to see if config is available during loading time of externalDependencies.

**Comment by PonteIneptique** [comment]
I think it's quite important for dagre :)

**Comment by fbaumgardt** [comment]
Yep! The issue was that this specific version was not available on CDN and so I linked a local copy. Sorry, forgot about the hardcode! :\

**Comment by PonteIneptique** [comment]
The only two I had to rewrite where i18n and dagre : 
```javascript
window.i18npath = "{{url_for(".r_arethusa_assets", filename="i18n/")}}";
window.dagred3path = "vendor/dagre-d3.min.js";
```
